### ISSUE TITLE
`ogc-example` project is tough to get running

### ISSUE THREAD

**Issue by notthatbreezy** [geotrellis/geotrellis-server#164]
The `ogc-example` project's application [conf](https://github.com/geotrellis/geotrellis-server/blob/develop/ogc-example/src/main/resources/application.conf) includes resources that are not publicly accessible or easily reconfigured.

I think an improvement would be to use `CogNode`'s instead of GeoTrellis layers (even if the resources are still not publicly accessible) since those can at least be more easily substituted in a development environment. This scenario happens sometimes when I've run into bugs/issues with some of the OGC services and had to figure out if the bug existed in my application code or exists in the library -- being able to quickly come up with a proof-of-concept that either reproduces or fails to reproduce the problem can be a blocker in that workflow.

**Comment by echeipesh** [comment]
Would it be more helpful to have an easy way to standup an instance of a given service from console for this kind of thing?

For instance:
```scala
val service = WcsService.fromMap(Map("layer1" -> GeoTiffRasterSource(???), "layer2" -> GeoTrellisRasterSource(???))

service.bind("0.0.0.0") // ??? --some kind of one command utility to standup an endpoint
```

**Comment by notthatbreezy** [comment]
I think that could work, but that's not what I had in mind.

I was thinking the conf idea is still a good one, but having one that is more accessible and easier to get started with maybe a few different examples - and starting the server with one as an argument. Right now there is a lot to edit in the existing example conf and it's not even really possible to switch to using a COG rather than a GT source without making code changes.

**Comment by echeipesh** [comment]
I agree with all of that, lets keep this issue specifically for that conf cleanup and breakup work.
